snap/hooks/post-refresh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# https://github.com/canonical/steam-snap/issues/19#issuecomment-1171984524
+# https://gist.github.com/jhenstridge/425c769949d034033f3d5d90acc2f181
+
+set -e
+
+mkdir -p $SNAP_DATA/etc
+
+snap_ld_so_conf=$SNAP_DATA/etc/snap-ld.so.conf
+snap_ld_so_cache=$SNAP_DATA/etc/snap-ld.so.cache
+real_ld_so_cache=$SNAP_DATA/etc/ld.so.cache
+
+RUNTIME=$SNAP/gnome-platform
+
+cat > "$snap_ld_so_conf" <<EOF
+/var/lib/snapd/lib/gl
+/var/lib/snapd/lib/gl32
+$SNAP/lib/x86_64-linux-gnu
+$SNAP/lib/i386-linux-gnu
+$SNAP/lib
+$SNAP/usr/lib/x86_64-linux-gnu
+$SNAP/usr/lib/i386-linux-gnu
+$SNAP/usr/lib
+$SNAP/usr/lib/x86_64-linux-gnu/dri
+$SNAP/usr/lib/i386-linux-gnu/dri
+$SNAP/usr/lib/i386-linux-gnu/pulseaudio
+$RUNTIME/lib/x86_64-linux-gnu
+$RUNTIME/lib/i386-linux-gnu
+$RUNTIME/lib
+$RUNTIME/usr/lib/x86_64-linux-gnu
+$RUNTIME/usr/lib/i386-linux-gnu
+$RUNTIME/usr/lib
+$RUNTIME/usr/lib/x86_64-linux-gnu/dri
+$RUNTIME/usr/lib/i386-linux-gnu/dri
```

Comment:
These lines can be removed, since this snap isn't using the gnome platform snap.